### PR TITLE
Sjosey fix python payment

### PR DESCRIPTION
**Pull request by StephenJosey**
Not really sure if this is a "good" way to use Python (throwing the whole HTML page in a Python script). I couldn't find an easer way to prevent replacing credentials in multiple places. If preferred, I can bring back "REPLACE_ME" and remove `config.ini`, so that I can move the `index.py` back into `index.html`.

### CODE REVIEW

File: connect-examples/v2/python_payment/cgi-bin/process_card.py [hunk 1]
```diff
@@ -3,27 +3,41 @@
...
+if config.get("DEFAULT", "is_prod") == "true":
+    access_token = config.get("PRODUCTION", "access_token")
+else:
+    access_token = config.get("SANDBOX", "access_token")
+
+squareconnect.configuration.access_token = access_token
 
 # The ID of the business location to associate processed payments with.
 # See [Retrieve your business's locations]
 # (https://docs.connect.squareup.com/articles/getting-started/#retrievemerchantprofile)
```
Comment (shaofu88) [inline connect-examples/v2/python_payment/cgi-bin/process_card.py]: I would rewrite two if blocks something like below.
```
config_type = "PRODUCTION" if config.get("DEFAULT", "is_prod") == "true" else "SANDBOX"
access_token = config.get(config_type, "access_token")
location_id = config.get(config_type, "location_id")
```

File: connect-examples/v2/python_payment/sqpaymentform.js [hunk 2]
```diff
@@ -13,6 +7,7 @@ var locationId = "REPLACE_ME";
  * Modifying this function is not required, but can be customized if you
  * wish to take additional action when the form button is clicked.
  */
+
```
Comment (shaofu88) [inline connect-examples/v2/python_payment/sqpaymentform.js]: nit: extra line here

File: connect-examples/v2/python_payment/sqpaymentform.js [hunk 3]
```diff
@@ -1,9 +1,3 @@
-// Set the application ID
-var applicationId = "REPLACE_ME";
-
-// Set the location ID
-var locationId = "REPLACE_ME";
-
 /*
```
Comment (shaofu88) [inline connect-examples/v2/python_payment/sqpaymentform.js]: just curious is this file `sqpaymentform.js` the same as `php_payment/sqpaymentform.js`?

File: connect-examples/v2/python_payment/cgi-bin/index.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+# coding: utf-8
+import ConfigParser
+
+# To read your secret credentials
+config = ConfigParser.ConfigParser()
+config.read('config.ini')
+
+
+# Retrive credentials based on is_prod
+if config.get("DEFAULT", "is_prod") == "true":
```
Comment (shaofu88) [inline connect-examples/v2/python_payment/cgi-bin/index.py]: see my comment in `process_card.py`

File: connect-examples/v2/python_payment/sqpaymentform.js [hunk 5]
```diff
@@ -25,174 +20,181 @@ function requestCardNonce(event) {
...
-      }
-    },
+        /*
+         * callback function: inputEventReceived
+         * Triggered when: visitors interact with SqPaymentForm iframe elements.
+         */
+        inputEventReceived: function (inputEvent) {
+            switch (inputEvent.eventType) {
+                case 'focusClassAdded':
+                    /* HANDLE AS DESIRED */
```
Comment (hukid) [inline connect-examples/v2/python_payment/sqpaymentform.js]: The comment style seems not very consistent. You use /* */ here and '//' other places. Is this something by design?

File: connect-examples/v2/python_payment/sqpaymentform.js [hunk 6]
```diff
@@ -25,174 +20,181 @@ function requestCardNonce(event) {
...
+         * callback function: cardNonceResponseReceived
+         * Triggered when: SqPaymentForm completes a card nonce request
+         */
+        cardNonceResponseReceived: function (errors, nonce, cardData) {
+            if (errors) {
+                // Log errors from nonce generation to the Javascript console
+                console.log("Encountered errors:");
+                errors.forEach(function (error) {
+                    console.log('  ' + error.message);
+                    alert(error.message);
```
Comment (hukid) [inline connect-examples/v2/python_payment/sqpaymentform.js]: I don't think alert for each error is a good experience. Can we make this a better error handling example?

File: connect-examples/v2/python_payment/sqpaymentform.js (hunk 5 above)
Comment (StephenJosey) [reply connect-examples/v2/python_payment/sqpaymentform.js]: This is pulled from our SqPaymentForm.js on our Doc site. If it needs to be changed, it should be changed wherever that comes from. I can change on mine, though (I would need to change the rest of the examples as well).

File: connect-examples/v2/python_payment/sqpaymentform.js (hunk 6 above)
Comment (StephenJosey) [reply connect-examples/v2/python_payment/sqpaymentform.js]: You're right, I'll remove the alert. I also have it outlining the text boxes in red, with a message "Please fix all errors." Should I add a special error for each one? (I would need to go back and change this on all examples as well.)

File: connect-examples/v2/python_payment/sqpaymentform.js (hunk 5 above)
Comment (StephenJosey) [reply connect-examples/v2/python_payment/sqpaymentform.js]: Oh wait. /* is used for multiline comments, // is used for single line comments. I didn't completely understand, but I see what you mean now.

File: connect-examples/v2/python_payment/cgi-bin/process_card.py (hunk 1 above)
Comment (StephenJosey) [reply connect-examples/v2/python_payment/cgi-bin/process_card.py]: Updated to that style.

File: connect-examples/v2/python_payment/sqpaymentform.js (hunk 2 above)
Comment (StephenJosey) [reply connect-examples/v2/python_payment/sqpaymentform.js]: Removed.

File: connect-examples/v2/python_payment/cgi-bin/index.py (hunk 4 above)
Comment (StephenJosey) [reply connect-examples/v2/python_payment/cgi-bin/index.py]: Updated.

**Pull request merged**